src/patternfly/components/LabelGroup/examples/LabelGroup.md

```diff
@@ -514,7 +514,7 @@ In addition to the JavaScript management of [editable labels](/components/label#
         {{/label}}
       {{/label-group-list-item}}
       {{#> label-group-list-item}}
-        {{#> label label--modifier="pf-m-blue pf-m-active" label--id=(concat label-group--id '-editable-label-active') label--isRemovable="true" label--IsEditable="true" label--IsEditableActive="true"}}
+        {{#> label label--modifier="pf-m-blue pf-m-active" label--id=(concat label-group--id '-editable-label-active') label--IsEditable="true" label--IsEditableActive="true"}}
```

Comment:
It looks like the `pf-m-active` class isn't doing anything for the label component here. In React, that class doesn't get applied anywhere for Label or LabelGroup. Same would apply to line 583. Should this class be something to add to the React implementation, or removed from Core?

Reply:
Nice find! Looks like it was added here by accident, it doesn't seem to do anything. https://github.com/patternfly/patternfly/pull/4097/files